webview/src/experiments/components/table/ExperimentCell.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import cx from 'classnames'
+import { VSCodeTag } from '@vscode/webview-ui-toolkit/react'
+import { CommitData } from 'dvc/src/experiments/webview/contract'
+import styles from './styles.module.scss'
+import Tooltip from '../../../shared/components/tooltip/Tooltip'
+import { Icon } from '../../../shared/components/Icon'
+import { GitCommit } from '../../../shared/components/icons'
+
+const CommitCell: React.FC<{
+  commit: CommitData
+  description: string
+  label: string
+  sha?: string
+}> = ({ commit, description, label, sha }) => {
+  const { tags, author, message, date } = commit
+  const tooltipContent = (
+    <div className={styles.experimentCellSecondaryNameTooltip}>
+      <div>
+        <p className={styles.sha}>
+          <Icon width={16} height={16} icon={GitCommit} /> {sha?.slice(0, 7)}
+        </p>
+        {tags.length > 0 && (
+          <>
+            {tags.map(tag => (
+              <VSCodeTag className={styles.tag} key={tag}>
+                {tag}
+              </VSCodeTag>
+            ))}
+          </>
+        )}
+      </div>
+      <p>
+        {author}, {date}
+      </p>
+      <p className={styles.message}>{message}</p>
+    </div>
+  )
+
+  return (
+    <Tooltip
+      placement="bottom-start"
+      appendTo={document.body}
+      content={tooltipContent}
+    >
+      <div className={styles.experimentCellText}>
+        <span>{label}</span>
+        <br />
```

Comment:
Can these be two `<div> tags or have `display: block;` on them instead of using a `<br />`. It's usually best to avoid `<br />` tags because it isn't semantic.